contents/blog/posthog-vs-pendo.md

```diff
@@ -0,0 +1,651 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Analyze user and revenue retention</td>
+        </tr>
+        <tr>
+            <td><strong>UTM tracking</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track marketing campaigns with UTM tags</td>
+        </tr>
+        <tr>
+            <td><strong>Correlation analysis</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Suggested events and properties that lead to success or failure</td>
+        </tr>
+        <tr>
+            <td><strong>Group analytics</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Understand how organizations use your product</td>
+        </tr>
+        <tr>
+            <td><strong>Formulas</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Use custom formulas to calculate unique insights</td>
+        </tr>
+        <tr>
+            <td><strong>SQL access</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Write your own queries in SQL</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **Correlation analysis:** [Correlation analysis](/product-analytics) automatically highlights significant linking properties or events relevant to an insight. For example, you can find if users who complete a funnel are likely to be from a certain location, or have completed another event. In PostHog, correlation analysis is a standard part of analytics. 
+
+- **Formulas:** Formulas in PostHog enable you to create custom insights using your data. A simple example of a formula would be an equation to figure out a ratio or percentage (e.g. the percentage of user who completed two different events), though advanced formulas can use more elaborate functions, such as `COS` and `SIN`.
+
+- **SQL access:** While both PostHog and Pendo have ready-made insights and visualization types, only PostHog gives you unlimited access to your data by [writing your own SQL queries](/docs/product-analytics/hogql). This is ideal for data scientists, product managers, and engineers who want to perform advanced analysis on user data.
+
+<h4 className="mb-0">Discover what's possible with <span className="text-red">product analytics</span></h4>
```

Comment:
```suggestion
<h4 className="mb-0">Discover what's possible with <span className="text-blue">product analytics</span></h4>
```
It is very jarring that this isn't a link but wherever we use `text-red `it is a link. Blue is the only other colour we don't use.